test/reline/yamatanooroti/multiline_repl

```diff
@@ -59,8 +59,24 @@ opt.on('--dynamic-prompt-show-line') {
     }
   }
 }
+
+def assert_auto_indent_params(lines, line_index, byte_pointer, is_newline)
+  raise 'Wrong lines type' unless lines.all? { |l| l.is_a? String }
+  raise 'Wrong line_index value' unless line_index.is_a?(Integer) && (0...lines.size).cover?(line_index)
```

Comment:
How about we simply check if `line` is present after `line = lines[line_index]`?

Reply:
I want to reject negative value, but maybe it's too much. I'll use line presence

Reply:
I see. Yeah there indeed is a risk of having negative values here. Like if the [outer condition](https://github.com/ruby/reline/blob/0924f2a075707ecf1061eab78844d036d44f5e39/lib/reline/line_editor.rb#L1645) is not correct, we could get that [here](https://github.com/ruby/reline/blob/0924f2a075707ecf1061eab78844d036d44f5e39/lib/reline/line_editor.rb#L1654).

I will let you decide whether to add a negative value check 👍

Reply:
I want to go with the simple one, no negative check for now.
It might be negative and it's safer, but the big problem in line_editor is mostly overrun of byte_pointer, I think.